docs/provider/mainnet-provider-setup.md

```diff
@@ -0,0 +1,707 @@
+
+:::info 
+Note that this TX can also be used to increase the provider's stake. To increase, the amount should be the sum of the current amount and the desired addition. For example, a provider is staked with `100ulava` and wants to increase its stake by `10ulava`. Then, the appropriate `amount` argument for the `stake-provider` command should be `110ulava`.
+:::
+
+:::info 
+Use the optional `--provider` flag to define a different address that will be used to operate the provider. This will make the address of the `--from` flag to be the provider's vault address, which will hold the provider's funds and will be able to perform stake or funds related transactions (like unstake, modify-stake and more).
+:::
+
+#### Parameters Description
+
+- **`chain-id`** - The ID of the serviced chain (e.g., **`COS4`** or **`FTM250`**). To get the full list of available chains use: `lavap query spec show-all-chains --node {LAVA_RPC_NODE}`.
+- **`amount`** - Stake amount for the specific chain (e.g., **`2010ulava`**).
+- **`endpoint`** - Provider host listener, composed of `provider-host:provider-port,geolocation`. To define multiple endpoints do: `"host:port,geolocation host2:port2,geolocation2"`. To define advanced endpoint features like `trace`, `debug`, and more, ask the Lava team.
+- **`geolocation`** - Indicates the geographical location where the process is located (e.g., **`1`** for US or **`2`** for EU). This argument must specify all the geolocations of the defined endpoints (see the "Ethereum Mainnet in US And Europe" below).
+- **`validator`** - Validator address for staking (optional). Validator address is required when staking due to the dualstaking mechanism. If no validator is specified, the highest stake validator will be picked automatically. Note that the validator address should be in a sdk.ValAddress format (`lava@valoper...`).
+
+#### Geolocations
+
+```javascript    
+    USC = 1; // US-Center
+    EU = 2; // Europe
+    USE = 4; // US-East
+    USW = 8; // US-West
+    AF = 16; // Africa
+    AS = 32; // Asia
+    AU = 64;  // (Australia, includes NZ)
+    GL = 65535; // Global
+```
+
+
+#### Flags Details
+
+- **`--from`** - The account to be used for the provider staking (e.g., **`my_account`**).
+- **`--provider-moniker`** - Provider’s public name
+- **`--keyring-backend`** - A keyring-backend of your choosing (e.g., **`test`**).
+- **`--chain-id`** - The chain_id of the network (e.g., **`lava-mainnet-1`**).
+- **`--gas`** - The gas limit for the transaction (e.g., **`"auto"`**).
+- **`--gas-adjustment`** - The gas adjustment factor (e.g., **`"1.5"`**).
+- **`--node`** - A RPC node for Lava (Official mainnet RPC will be added soon).
+- **`--provider`** - The provider's operational address (address used to operate the provider process, default is vault address).
+
+### Stake Examples
+
+#### Lava Testnet in US (with specified validator)
```

Comment:
testnet -> mainnet

Reply:
Solved in new commit https://github.com/silentnoname/lavadocs/commit/d2d639ebcf4b8f5d49b21bfee37c605e20a3c345